{"uid":"2d2a492ec83a0b6f","name":"User is not allowed to enter invalid phone number","fullName":"tests.test_login.TestLogin#test_invalid_phone_number","historyId":"ce62b6e95f8e8da009fe6f309ba4fc9a","time":{"start":1683297325882,"stop":1683297325882,"duration":0},"description":"User is not allowed to enter invalid phone number","descriptionHtml":"<p>User is not allowed to enter invalid phone number</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x561b4cdd4fe3 <unknown>\n#1 0x561b4cb13d36 <unknown>\n#2 0x561b4cb50c4d <unknown>\n#3 0x561b4cb50d61 <unknown>\n#4 0x561b4cb8c6a4 <unknown>\n#5 0x561b4cb718ed <unknown>\n#6 0x561b4cb8a232 <unknown>\n#7 0x561b4cb71693 <unknown>\n#8 0x561b4cb4403a <unknown>\n#9 0x561b4cb4517e <unknown>\n#10 0x561b4cd96dbd <unknown>\n#11 0x561b4cd9ac6c <unknown>\n#12 0x561b4cda44b0 <unknown>\n#13 0x561b4cd9bd63 <unknown>\n#14 0x561b4cd6ec35 <unknown>\n#15 0x561b4cdbf138 <unknown>\n#16 0x561b4cdbf2c7 <unknown>\n#17 0x561b4cdcd093 <unknown>\n#18 0x7fc762f64b43 <unknown>","statusTrace":"request = <SubRequest 'setup' for <Function test_invalid_phone_number>>\n\n    @pytest.fixture(scope=\"function\")\n    def setup(request:pytest.FixtureRequest):\n        browser=request.config.getoption(\"--browser\")\n        if(browser in(\"chrome\",\"chrome_headless\")):\n            chrome_options = webdriver.ChromeOptions()\n            chrome_options.set_capability(\n                    \"goog:loggingPrefs\", { \"browser\": \"ALL\"}\n                )\n    \n        match browser:\n            case \"chrome\":\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"chrome_headless\":\n                chrome_options.add_argument(\"headless=new\")\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"firefox\":\n                driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))\n        driver.get(\"https://sporty.com/news/latest\")\n        # make sure the page is loaded\n        wait= WebDriverWait(driver,30)\n>       wait.until(expected_conditions.visibility_of_element_located(((By.XPATH,\"//*[contains(text(),'Customise your')]\"))))\n\ntests/conftest.py:43: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"659e7b0ff241b95d6e19966bbe9ed3b1\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f2a374e9510>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x561b4cdd4fe3 <unknown>\nE       #1 0x561b4cb13d36 <unknown>\nE       #2 0x561b4cb50c4d <unknown>\nE       #3 0x561b4cb50d61 <unknown>\nE       #4 0x561b4cb8c6a4 <unknown>\nE       #5 0x561b4cb718ed <unknown>\nE       #6 0x561b4cb8a232 <unknown>\nE       #7 0x561b4cb71693 <unknown>\nE       #8 0x561b4cb4403a <unknown>\nE       #9 0x561b4cb4517e <unknown>\nE       #10 0x561b4cd96dbd <unknown>\nE       #11 0x561b4cd9ac6c <unknown>\nE       #12 0x561b4cda44b0 <unknown>\nE       #13 0x561b4cd9bd63 <unknown>\nE       #14 0x561b4cd6ec35 <unknown>\nE       #15 0x561b4cdbf138 <unknown>\nE       #16 0x561b4cdbf2c7 <unknown>\nE       #17 0x561b4cdcd093 <unknown>\nE       #18 0x7fc762f64b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1683297325883,"stop":1683297367275,"duration":41392},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x561b4cdd4fe3 <unknown>\n#1 0x561b4cb13d36 <unknown>\n#2 0x561b4cb50c4d <unknown>\n#3 0x561b4cb50d61 <unknown>\n#4 0x561b4cb8c6a4 <unknown>\n#5 0x561b4cb718ed <unknown>\n#6 0x561b4cb8a232 <unknown>\n#7 0x561b4cb71693 <unknown>\n#8 0x561b4cb4403a <unknown>\n#9 0x561b4cb4517e <unknown>\n#10 0x561b4cd96dbd <unknown>\n#11 0x561b4cd9ac6c <unknown>\n#12 0x561b4cda44b0 <unknown>\n#13 0x561b4cd9bd63 <unknown>\n#14 0x561b4cd6ec35 <unknown>\n#15 0x561b4cdbf138 <unknown>\n#16 0x561b4cdbf2c7 <unknown>\n#17 0x561b4cdcd093 <unknown>\n#18 0x7fc762f64b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/selenium-python/selenium-python/tests/conftest.py\", line 43, in setup\n    wait.until(expected_conditions.visibility_of_element_located(((By.XPATH,\"//*[contains(text(),'Customise your')]\"))))\n  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"testStage":{"description":"User is not allowed to enter invalid phone number","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x561b4cdd4fe3 <unknown>\n#1 0x561b4cb13d36 <unknown>\n#2 0x561b4cb50c4d <unknown>\n#3 0x561b4cb50d61 <unknown>\n#4 0x561b4cb8c6a4 <unknown>\n#5 0x561b4cb718ed <unknown>\n#6 0x561b4cb8a232 <unknown>\n#7 0x561b4cb71693 <unknown>\n#8 0x561b4cb4403a <unknown>\n#9 0x561b4cb4517e <unknown>\n#10 0x561b4cd96dbd <unknown>\n#11 0x561b4cd9ac6c <unknown>\n#12 0x561b4cda44b0 <unknown>\n#13 0x561b4cd9bd63 <unknown>\n#14 0x561b4cd6ec35 <unknown>\n#15 0x561b4cdbf138 <unknown>\n#16 0x561b4cdbf2c7 <unknown>\n#17 0x561b4cdcd093 <unknown>\n#18 0x7fc762f64b43 <unknown>","statusTrace":"request = <SubRequest 'setup' for <Function test_invalid_phone_number>>\n\n    @pytest.fixture(scope=\"function\")\n    def setup(request:pytest.FixtureRequest):\n        browser=request.config.getoption(\"--browser\")\n        if(browser in(\"chrome\",\"chrome_headless\")):\n            chrome_options = webdriver.ChromeOptions()\n            chrome_options.set_capability(\n                    \"goog:loggingPrefs\", { \"browser\": \"ALL\"}\n                )\n    \n        match browser:\n            case \"chrome\":\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"chrome_headless\":\n                chrome_options.add_argument(\"headless=new\")\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"firefox\":\n                driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))\n        driver.get(\"https://sporty.com/news/latest\")\n        # make sure the page is loaded\n        wait= WebDriverWait(driver,30)\n>       wait.until(expected_conditions.visibility_of_element_located(((By.XPATH,\"//*[contains(text(),'Customise your')]\"))))\n\ntests/conftest.py:43: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"659e7b0ff241b95d6e19966bbe9ed3b1\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f2a374e9510>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x561b4cdd4fe3 <unknown>\nE       #1 0x561b4cb13d36 <unknown>\nE       #2 0x561b4cb50c4d <unknown>\nE       #3 0x561b4cb50d61 <unknown>\nE       #4 0x561b4cb8c6a4 <unknown>\nE       #5 0x561b4cb718ed <unknown>\nE       #6 0x561b4cb8a232 <unknown>\nE       #7 0x561b4cb71693 <unknown>\nE       #8 0x561b4cb4403a <unknown>\nE       #9 0x561b4cb4517e <unknown>\nE       #10 0x561b4cd96dbd <unknown>\nE       #11 0x561b4cd9ac6c <unknown>\nE       #12 0x561b4cda44b0 <unknown>\nE       #13 0x561b4cd9bd63 <unknown>\nE       #14 0x561b4cd6ec35 <unknown>\nE       #15 0x561b4cdbf138 <unknown>\nE       #16 0x561b4cdbf2c7 <unknown>\nE       #17 0x561b4cdcd093 <unknown>\nE       #18 0x7fc762f64b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[],"attachments":[{"uid":"dc4aeb1c28cee6b0","name":"stderr","source":"dc4aeb1c28cee6b0.txt","type":"text/plain","size":159}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"epic","value":"Login functionality"},{"name":"story","value":"Login functionality"},{"name":"tag","value":"@pytest.mark.usefixtures('setup')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az453-30"},{"name":"thread","value":"2161-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["@pytest.mark.usefixtures('setup')"]},"source":"2d2a492ec83a0b6f.json","parameterValues":[]}